IndoorMapsFrontend/src/components/pageSections/style/FixedFooter.css

```diff
@@ -0,0 +1,17 @@
+.directoryMain {
+    margin-bottom: 5em;
+}
+
+footer.notFlexboxPage  {
```

Comment:
This css class still seems to be using flex property? The name is little confusing

Reply:
Change Applied: https://github.com/Theo-s-Meta-Capstone/IndoorMaps/commit/0a21c4bf5e7cdcd4168561688286076a77c660df